tests/test_candles_fix.py

```
@@ -4,159 +4,165 @@
+        print(f"Test failed with error: {e}")
         import traceback
+
         traceback.print_exc()
-    
+
     finally:
         try:
             await client.disconnect()
-            print("\n🔌 Disconnected from PocketOption")
+            print("\nDisconnected from PocketOption")
         except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)